### PR TITLE
Added time-to-live support

### PR DESCRIPTION
**Pull request by mabott**
Moved host/user/pass to top. Added optional TTL argument support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,13 @@ Once your script is working, just add a reference to the script to crontab or ot
 
 Please note that as of version 2.1.6/2.1.7 we only support "/" as the source path for snapshots.
 
+As of version 2.5.1 we support snapshot expiration. You can supply a single argument to create_snapshot.sh:
+
+    crontab -e
+    0 9,18 * * Mon  /home/admin/scripts/create_snapshot.sh 30d
```
Comment (mabott) [inline README.md]: `mbottMBP:snapshot_scripts mbott$ ./create_snapshot.sh -t 10d
{
    "created_by_policy": false, 
    "directory_name": "6_20161221142846", 
    "expiration": "2016-12-31T22:28:46.438709Z", 
    "id": 6, 
    "name": "20161221142846", 
    "source_path": "/", 
    "timestamp": "2016-12-21T22:28:46.640635665Z"
}
mbottMBP:snapshot_scripts mbott$ ./create_snapshot.sh --timetolive 10d
{
    "created_by_policy": false, 
    "directory_name": "7_20161221142851", 
    "expiration": "2016-12-31T22:28:52.263611Z", 
    "id": 7, 
    "name": "20161221142851", 
    "source_path": "/", 
    "timestamp": "2016-12-21T22:28:52.465231848Z"
}
mbottMBP:snapshot_scripts mbott$`

Comment (mabott) [inline README.md]:
```
mbottMBP:snapshot_scripts mbott$ ./create_snapshot.sh --fakeoption 1
Unknown option --fakeoption
mbottMBP:snapshot_scripts mbott$ 
```

**Comment by MichaelSEA** [comment]
is there a reason we'd call it --timetolive/ -t rather than --expires?  I know they're almost synonymous, but TTL is a network thing.  Thoughts?

**Comment by mabott** [comment]
Heh, I didn't make up the name of that option on my own! I was being consistent with qq. "expires" is an absolute date, whereas "timetolive" is a relative date.

```
mbottMBP:8_edelbrock mbott$ qq snapshot_create_snapshot --help
usage: qq snapshot_create_snapshot [-h] [-n NAME]
                                   [-e EXPIRATION | -t TIMETOLIVE]

Create a new snapshot.

optional arguments:
  -h, --help            show this help message and exit
  -n NAME, --name NAME  Name of the snapshot.
  -e EXPIRATION, --expiration EXPIRATION
                        Time at which to expire the snapshot. Providing an
                        empty string or omitting this argument indicates the
                        snapshot should never be expired. Format according to
                        RFC 3339, which is a normalized subset of ISO 8601.
                        See http://tools.ietf.org/rfc/rfc3339.txt, section 5.6
                        for ABNF.
  -t TIMETOLIVE, --timetolive TIMETOLIVE
                        Time to live for the snapshot in the form '#D' where #
                        is a positive number and D specifies the unit for #.
                        Choices for D are w -> weeks, d -> days, h -> hours,
                        and m -> minutes.
mbottMBP:8_edelbrock mbott$ 
```

**Pull request merged**